infra/modules/providers/azure/container-registry/output.tf

```diff
@@ -13,3 +13,12 @@ output "container_registry_name" {
   value       = var.container_registry_name
 }
 
+output "admin_username" {
+  description = "If admin access is enabled, this will be the username for the ACR"
+  value = azurerm_container_registry.container_registry.admin_username
+}
+
+output "admin_password" {
```

Comment:
We should avoid exporting sensitive values like passwords and private keys from our modules.

Reply:
Is there a way to export this as a password... ie *************

Reply:
This information is stored in the state file either way. You can mark the output as sensitive by using a `sensitive   = true` flag, which I'll add. This only obfuscates the value in CLI output.

Reply:
When I made my earlier comment, I was unaware that exported TF data source attributes are saved in the state files. Even more shocked that the password attribute for the ACR data source provider isn't marked as sensitive. Your approach makes sense. Thanks for being so thorough with validating the behavior for both approaches.